docs/SPEC.md

```diff
@@ -2067,34 +2067,43 @@ from(bucket:"telegraf/autogen")
 Group groups records based on their values for specific columns.
 It produces tables with new group keys based on the provided properties.
 
-Group has the following properties:
+Group accepts _exactly one_ of the following properties:
 
 *  `by` list of strings
     Group by these specific columns.
-    Cannot be used with `except`.
 *  `except` list of strings
     Group by all other columns except this list.
-    Cannot be used with `by`.
+*  `none` boolean
+    Group by nothing, so, group all records into a single group.
+*  `all` boolean
+    Do not group.
+    
+Specifying more than one of the properties above would result in an error.
 
 Examples:
-    group(by:["host"]) // group records by their "host" value
-    group(except:["_time", "region", "_value"]) // group records by all other columns except for _time, region, and _value
-    group(by:[]) // group all records into a single group
-    group(except:[]) // group records into all unique groups
+
+*    `group(by: ["host"])`: group records by their `"host"` value.
+*    `group(except: ["_time", "region", "_value"])`: group records by the values in all other columns except for
+     `"_time"`, `"region"`, and `"_value"`.
+*    `group(none: true)`: group all records into a single group
+*    `group(all: true)`: ignores grouping. Equivalent to `group()`.
```

Comment:
`group()` Should be invalid.

Reply:
Indeed it works, it defaults to `group(all: true)`: https://github.com/influxdata/flux/blob/master/functions/transformations/group.go#L80

I can avoid the example in the SPEC.

Reply:
@affo that is a bug. Good find. Lets remove it from the SPEC examples and file a bug on the repo.